docs/contributing/Pull_Requests.mdx

```diff
@@ -74,4 +74,10 @@ We generally review PRs oldest to newest, unless we consider a newer PR higher p
 
 Once we have reviewed your PR, we will provide any feedback that needs addressing.
 If you feel a requested change is wrong, don't be afraid to discuss with us in the comments.
-Once the feedback is addressed, and the PR is reviewed, we'll ensure the branch is up to date with `main`, and merge it for you.
+
+Please post comments as [line comments](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) when possible, so that they can be threaded.
+You can [resolve conversations](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#resolving-conversations) on your own when you feel they're resolved - no need to comment explicitly and/or wait for a maintainer.
```

Comment:
@IronGeek does this answer what you were looking for?

Reply:
@JoshuaKGoldberg Yes, thank you :pray: 
I'm always not sure if I should resolve the conversation myself or wait for the reviewer, this surely clears things up.